appsync-cognito-lambda-cdk/README.md

```diff
@@ -12,80 +13,68 @@ Important: this application uses various AWS services and there are costs associ
 - [Git Installed](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
-- [AWS Serverless Application Model](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/serverless-sam-cli-install.html) (AWS SAM) installed
+- [AWS Cloud Development Kit](https://docs.aws.amazon.com/cdk/api/v2/) (AWS CDK) installed
 
 ## Deployment Instructions
 
-1. Create a new directory, navigate to that directory in a terminal and clone the GitHub repository:
+1. Clone the ServerlessLand repository
    ```
    git clone https://github.com/aws-samples/serverless-patterns
    ```
 2. Change directory to the pattern directory:
+
    ```
    cd ./serverless-patterns/cognito-appsync-lambda-cdk/
    ```
-3. Navigate to `/cognito-appsync-lambda-cdk/bin/cognito-auth-cdk.ts` and change the `account` and `region` environment variables to suit your values.
-
-```typescript
-env: { account: "13xxxxxxxxxx", region: "us-east-2" },
-```
 
-5. From the command line, use AWS CDK to deploy the AWS resources for the pattern as specified in the `./cognito-appsync-lambda-cdk/lib/cognito-auth-cdk-stack.ts` file:
+3. From the command line, use AWS CDK to deploy the AWS resources for the pattern as specified in the `./cognito-appsync-lambda-cdk/lib/cognito-auth-cdk-stack.ts` file:
 
    ```bash
-   cdk deploy
+   npx aws-cdk deploy
    ```
 
 ## Testing
 
-Sign in to your AWS console and search for appsync. Open up appsync and click on your newly deployed project.
-
-- On the left hand side menu, click on `Settings` and create a new `API_KEY`.
-- Click on `Queries` on the left hand side menu to create and run your mutation.
+When our application was deployed, the `UserpoolId` was output to our terminal. We'll use that value to create a user in our Userpool by running the following command:
 
-- Create your first mutation. See image below.
-  ![alt text](./assets/mutation_non_auth.png)
+```bash
+aws cognito-idp sign-up \
+  --client-id <client-id> \
+  --username <username> \
+  --password <password> \
+  --user-attributes Name=<attribute-name>,Value=<attribute-value>
+```
 
-- Run the mutation and you should get an `unauthorized access` error. See image below
-  ![alt text](./assets/mutation_non_auth_1.png)
+Where:
 
-This occurs because we applied the appsync directive `@aws_cognito_user_pools` to the mutation endpoint in `schema.graphql`.
-That appsync directive ensures that, that mutation can only be accessed by authenticated cognito users.
+- `client-id`: the app client ID of your user pool.
+- `username`: the desired username for the new user.
+- `password`: the password for the new user.
+- `attribute-name`: the name of the user attribute to set, such as email or phone_number.
+- `attribute-value`: the value to set for the user attribute.
 
-So let's go ahead and create a new user in cognito. See screenshots.
+Sign in to your AWS console and search for appsync. Open up appsync and click on your newly deployed project.
```

Comment:
before this is important that the user is authenticated if not this wont work. add that to the test.

Reply:
how you confirm your user using aws cli? Please add clear instructions